test/lib/code_corps/endpoint_test.ex

```diff
@@ -0,0 +1,31 @@
+defmodule CodeCorps.EndpointTest do
+  use CodeCorps.ConnCase
```

Comment:
This is great! I believe there's a `PageControllerTest` that's doing some CORS testing. Would you mind folding that into this test (or eliminating it, if you deem it necessary)?